src/content/containers/Home.js

```diff
@@ -2,74 +2,282 @@
+        </div>
+        <div className='homeInstructionsRight'>
+          <h2>Getting started</h2>
+          <p>
+            Install the Gecko Profiler Add-on to start recording a performance profile in
+            Firefox, then analyze it and share it with perf.html, or drag a saved profile
+            here to load it in perf.html.
+          </p>
+          <InstallButton name='Gecko Profiler' className='homeSectionInstallButton' xpiURL={ADDON_URL}>
+            <span className='homeSectionPlus'>+</span>
+            Install add-on
+          </InstallButton>
+        </div>
+      </div>
+    );
+  }
+
+  _renderRecordInstructions() {
+    return (
+      <div className='homeInstructions' key={1}>
+        <div className='homeInstructionsLeft'>
+          <p>
+            <img className='homeSectionScreenshot' src={AddonScreenshot} />
+          </p>
+        </div>
+        <div className='homeInstructionsRight'>
+          <h2>Recording profiles</h2>
+          <p>
+            To start profiling, click on the profiling button, or use the keyboard
+            shortcuts. The icon is blue when a profile is recording. Hit
+            <kbd>Capture Profile</kbd> to load the data into perf.html.
+          </p>
+          {this._renderShortcuts()}
+        </div>
+      </div>
+    );
   }
-  return 'https://raw.githubusercontent.com/devtools-html/Gecko-Profiler-Addon/master/gecko_profiler.xpi';
+
+  _renderLegacyInstructions() {
+    return (
+      <div className='homeInstructions' key={2}>
+        <div className='homeInstructionsLeft'>
+          <div style={{textAlign: 'center'}}>
+            <img className='homeSectionScreenshot' src={PerfScreenshot} />
+          </div>
+        </div>
+        <div className='homeInstructionsRight'>
+          <h2>Recording profiles</h2>
+          <p>
+            To start recording a performance profile in Firefox, first install the{' '}
+            <InstallButton name='Gecko Profiler' xpiURL={LEGACY_ADDON_URL}>
+              Gecko Profiler Add-on
+            </InstallButton>.
+            Then use the button added to the browser, or use the following shortcuts
+            to record a profile. The button's icon is blue when a profile is recording.
+            Hit <kbd>Capture Profile</kbd> to load the data into perf.html.
+          </p>
+          {this._renderShortcuts()}
+        </div>
+      </div>
+    );
+  }
+
+  _renderOtherBrowserInstructions() {
+    return (
+      <div className='homeInstructions' key={0}>
+        <div className='homeInstructionsLeft'>
+          <div style={{textAlign: 'center'}}>
+            <img className='homeSectionScreenshot' src={PerfScreenshot} />
+          </div>
+        </div>
+        <div className='homeInstructionsRight'>
+          <h2>How to view and record profiles</h2>
+          <p>
+            Recording performance profiles requires{' '}
+            <a href='https://www.mozilla.org/en-US/firefox/new/'>Firefox</a>, however
```

Comment:
How about this instead:

Recording performance profiles requires Firefox. However, existing profiles can be viewed in any modern browser. To view a profile, either follow a link to a public profile, or drag a saved profile onto this screen.